sbnana/SBNAna/Vars/NueVars.cxx

```diff
@@ -349,4 +349,45 @@ const Var kMuonTrackLength(
       }
       return length;
     });
+
+const Var kLongestTrackDazzlePID(
+    [](const caf::SRSliceProxy* slc) -> int {
+      int pdg(-5);
+      const int longestTrackIdx(kLongestTrackIdx(slc));
+      if (longestTrackIdx != -1) {
+        pdg = slc->reco.trk[longestTrackIdx].dazzle.pdg;
+      }
+      return pdg;
```

Comment:
I don't think the `pdg` variable is really necessary. How about `if(longestTrackIdx != -1) return slc->...; else return -5;`. Same for Vars below.